### PR TITLE
Newbranch test

### PR DESCRIPTION
**Pull request by RJDan**
I've provided some minor adjustments to the wording and work flow to make the content a bit easier for novices to follow.

### CODE REVIEW

**Comment by akshayparopkari** [comment]
Thanks for sending in the clarifications, @RJDan!

The changes seem good to me 👍 Before merging, it will be good to have a couple more sets of eyes on the proposed changes. What do you think, @datacarpentry/shell-genomics-maintainers?

**Review by akshayparopkari — APPROVED** [review]
Looks good to me 👍

**Review by aschuerch — APPROVED** [review]

**Comment by aschuerch** [comment]
thanks for the pull request @RJDan!

**Pull request merged**
